Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -456,7 +456,49 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
         ],
         handledNodes: [argument.id]
       )
-      return .visitChildren
+    }
+    if let unexpectedAtSign = node.unexpectedBeforeAtSignToken?.onlyToken(where: { $0.tokenKind == .atSign && !$0.trailingTrivia.isEmpty }) {
+      addDiagnostic(
+        unexpectedAtSign,
+        position: node.unexpectedBeforeAtSignToken?.endPositionBeforeTrailingTrivia,
```

Comment:
Oh, one thing that just occurred to me. We don’t actually need to optionally chaining because we know that `node.unexpectedBeforeAtSignToken` only contains `unexpectedAtSign`.

```suggestion
        position: unexpectedAtSign.endPositionBeforeTrailingTrivia,
```